ros/src/tl_detector/light_classification/test_tl_classifier.py

```diff
@@ -27,14 +27,16 @@ def setUp(self):
         TLClassifier.INSTANCE = None
         TLClassifier.KNOWN_TRAFFIC_LIGHT_CLASSIFIERS = {}
         TLClassifier.register_subclass("opencv")(OpenCVTLClassifier)
-        TLClassifier.register_subclass("ssd")(SSDTLClassifier)
+        TLClassifier.register_subclass("ssd-sim")(SSDSimTLClassifier)
+        TLClassifier.register_subclass("ssd-real")(SSDRealTLClassifier)
         TLClassifier.register_subclass("yolo-tiny")(YOLOTinyTLClassifier)
 
     def tearDown(self):
         TLClassifier.INSTANCE = None
         TLClassifier.KNOWN_TRAFFIC_LIGHT_CLASSIFIERS = {}
         TLClassifier.register_subclass("opencv")(OpenCVTLClassifier)
-        TLClassifier.register_subclass("ssd")(SSDTLClassifier)
+        TLClassifier.register_subclass("ssd-sim")(SSDSimTLClassifier)
+        TLClassifier.register_subclass("ssd-real")(SSDRealTLClassifier)
         TLClassifier.register_subclass("yolo-tiny")(YOLOTinyTLClassifier)
 
     def test_get_instance_of(self):
```

Comment:
This test will fail. Now you have 4 sub-classes instead of 3. To run this unit test execute: 
```
aoool@dell:~/Downloads/self-driving-car-using-ros/ros$ catkin_make
aoool@dell:~/Downloads/self-driving-car-using-ros/ros$ source devel/setup.bash 
aoool@dell:~/Downloads/self-driving-car-using-ros/ros$ cd src/tl_detector/
aoool@dell:~/Downloads/self-driving-car-using-ros/ros/src/tl_detector$ python -m unittest discover -s /home/aoool/Downloads/self-driving-car-using-ros/ros/src/tl_detector/light_classification -p test_tl_classifier.py -t /home/aoool/Downloads/self-driving-car-using-ros/ros/src/tl_detector/light_classification 
```

Reply:
Done